Test/dafny0/ParameterResolution.dfy

```diff
@@ -275,15 +275,15 @@ module NameOnlyParameters {
     least predicate LP(a: int, nameonly b: int, c: int := 100)
     static greatest predicate GP(a: int, nameonly b: int := 75, c: int := 100)
   }
-  iterator Iter(nameonly u: int, x: int)
+  iterator Iter(nameonly u: int, x: int) // error (x2, because of the desugaring of iterators): x is effectively nameonly, but not declared as such
```

Comment:
Maybe we should come up with a testing format where you can specify the expected error messages inline : )

Reply:
I do like being able to read the expected outcome in the test file itself. Therefore, I always put such "error" comments in the tests. But these comments aren't the exact error messages, don't capture column information, and in some cases are even placed only on a nearby line. It's plausible that we could turn these into a formally recognized testing file format.